### PR TITLE
Upgrade chaincode

### PR DESCRIPTION
**Pull request by sapthasurendran**
Signed-off-by: sapthasurendran <saptha.surendran@ibm.com>

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ import (
 
 const CmdRoot = "core"
 
+//starts here
```
Comment (adecaro) [inline cmd/node/main.go:20]: please remove this

File: integration/nwo/fabric/network/deploy.go [hunk 2]
```diff
@@ -68,7 +68,6 @@ func InstallChaincode(n *Network, chaincode *topology.Chaincode, peers ...*topol
 	if chaincode.PackageID == "" {
 		chaincode.SetPackageIDFromPackageFile()
 	}
-
```
Comment (adecaro) [inline integration/nwo/fabric/network/deploy.go:71]: no need to change this

File: integration/nwo/fabric/network/network.go [hunk 3]
```diff
@@ -259,8 +260,46 @@ func (n *Network) DeployChaincode(chaincode *topology.ChannelChaincode) {
 	if chaincode.Chaincode.InitRequired {
 		InitChaincode(n, chaincode.Channel, orderer, &chaincode.Chaincode, peers...)
 	}
+	//add new chaincode to the topology
+	n.topology.AddChaincode(chaincode)
```
Comment (adecaro) [inline integration/nwo/fabric/network/network.go:264]: why do we need this/

Comment (sapthasurendran) [reply integration/nwo/fabric/network/network.go:264]: Because after the chaincode upgrade, topology has to show the new version of the chaincode.

**Pull request merged**